7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        nomic_data.columns[nomic_data.columns.str.contains("[NSU]", regex=True)]
</pre><p>May consider setting this to a variable so you don't have this long expression each time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>